src/containers/RecordingsTable.tsx

```diff
@@ -104,17 +124,4 @@ const RecordingsTable: FunctionComponent<Props> = ({ recordings, onDeleteRecordi
     );
 }
 
-const mapStateToProps: MapStateToProps<StateProps, OwnProps, RootState> = (state: RootState, ownProps: OwnProps): StateProps => ({
-    recordings: state.recordings,
-    documentInfo: state.documentInfo
-})
-  
-const mapDispatchToProps: MapDispatchToProps<DispatchProps, OwnProps> = (dispatch: Dispatch<RootAction>, ownProps: OwnProps) => ({
-    onDeleteRecordings: (recordingIds: string[]) => deleteRecordings(dispatch, recordingIds),
-    onSetRecordingInfo: ({ recordingId, recordingInfo }) => dispatch(setRecordingInfo({ recordingId, recordingInfo }))
-})
-
```

Comment:
So you were able to remove the 'baseline' redux structure by explicitly creating the sorting and workspace reducers?

Reply:
yes